src/app/app.permissions.js

```diff
@@ -0,0 +1,18 @@
+angular.module('librarium').run(function (PermPermissionStore, PermRoleStore) {
+  PermPermissionStore.definePermission('loginOrSignup', function () {
+    return true;
+  });
+
+  PermPermissionStore.definePermission('noLoginOrSignup', function () {
```

Comment:
what are the permissions for? anonymous role is enough to secure login and signup endpoints